src/library_wasmfs.js

```diff
@@ -107,9 +107,12 @@ mergeInto(LibraryManager.library, {
       // TODO: Remove dependency on FS.cwd().
       // User code should not be using FS.cwd().
       // For file preloading, cwd should be '/' to begin with.
-      return '/';
+      return withStackSave(() => {
+        return __wasmfs_getcwd();
```

Comment:
Since this function is JS function that returns a string I don't think you want to just return a "char *" here since that will result in  Number (pointer) in JavaScript.

Reply:
@sbc100 I have declared the return type for __wasmfs_getcwd() as const char. Could you check the function singature and help me rectify it.